File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/sql/ConditionModel.java
package com.raizlabs.android.dbflow.test.sql;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.test.TestDatabase;
import com.raizlabs.android.dbflow.test.structure.TestModel1;

@Table(database = TestDatabase.class)
public class ConditionModel extends TestModel1 {
    @Column
    long number;

    @Column
    int bytes;

    @Column
    double fraction;

    @Column
    float floatie;

    @Column
    short shortie;

    @Column
    byte bytie;

    @Column
    char charie;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/sql/ConditionModel.kt
package com.raizlabs.android.dbflow.sql

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.TestDatabase
import com.raizlabs.android.dbflow.structure.TestModel1

@Table(database = TestDatabase::class)
class ConditionModel : TestModel1() {
    @Column
    var number: Long = 0

    @Column
    var bytes: Int = 0

    @Column
    var fraction: Double = 0.toDouble()

    @Column
    var floatie: Float = 0.toFloat()

    @Column
    var shortie: Short = 0

    @Column
    var bytie: Byte = 0

    @Column
    var charie: Char = ' '
}
